packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/components/PowerStateCellRenderer.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { TableCell } from 'src/modules/Providers/utils';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { VSphereVM } from '@kubev2v/types';
+import { Tooltip } from '@patternfly/react-core';
+import { OffIcon, PowerOffIcon, UnknownIcon } from '@patternfly/react-icons';
+
+import { VMCellProps } from './VMCellProps';
+
+export const PowerStateCellRenderer: React.FC<VMCellProps> = ({ data }) => {
```

Comment:
This renderer is specific to vmWarre, OCP status will may different ( it has more options that on/off )
If you choose to make a different status rendered for each type, this should have a unique name of path to indicate it's vmware onle.

For example if you choose unique name it can be VSpherePowerCellRendere, if you choose path it can be /cellRenderers/VSphere/powerCellRendere ?

Reply:
Switched to generic mapper borrowed from legacy package https://github.com/kubev2v/forklift-console-plugin/blob/8c17951019a08a33f55b7c820e694551dd99c6dc/packages/legacy/src/common/components/VMNameWithPowerState.tsx#L24